src/binary-reversal/index.js

```diff
@@ -3,6 +3,47 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  let stringToNumber = parseInt(value);
+  let decimalToBinary = stringToNumber.toString(2);
+
+  if (decimalToBinary.length < 8) {
+    let paddedBinary = "0000000" + stringToNumber.toString(2);
+    let actualBinary = paddedBinary.substring(
+      paddedBinary.length - 8,
+      paddedBinary.length
+    );
```

Comment:
Interesting. Look into the `padStart` method for string.

Reply:
> Interesting. Look into the `padStart` method for string.

okay... thanks